test/functional/patchinner.ts

```diff
@@ -129,11 +129,10 @@ describe('patching an element\'s children', () => {
   });
 
   it('should pass third argument to render function', () => {
-    function render(content:{}|undefined) {
-      text(content as string);
-    }
 
-    patchInner(container, render, 'foobar');
+    patchInner(container, () => {
```

Comment:
We shouldn't change this test, it is explicitly testing that the third argument to `patchInner` goes to the render function.

Reply:
Done